packages/authentication/src/__tests__/fixtures/keys.ts

```diff
@@ -0,0 +1,26 @@
+import {BindingKey} from '@loopback/context';
+import {BasicAuthenticationUserService} from './services/basic-auth-user-service';
+import {JWTService} from './services/jwt-service';
+import {UserRepository} from './users/user.repository';
+
+export const USER_REPO = BindingKey.create<UserRepository>(
+  'authentication.user.repo',
+);
+
+export namespace BasicAuthenticationStrategyBindings {
+  export const USER_SERVICE = BindingKey.create<BasicAuthenticationUserService>(
```

Comment:
```ts
  export const USER_SERVICE = BindingKey.create<UserService>(
    'services.authentication.basic.user.service',
  );

```
produces a typescript error most likely because `UserService` is defined like:
```ts
interface UserService<U, C>
```
In my test cases, I only user the `BasicAuthenticationUserService` for my basic authentication strategy mock implementation. I don't see a point in using `UserService` when creating a binding key.